httprequest_lego_provider/tests/test_views.py

```diff
@@ -264,3 +265,134 @@ def test_test_jwt_token_login(
         )
 
         assert response.status_code == 204
+
+
+@pytest.mark.django_db
+def test_post_domain_when_logged_in_as_non_admin_user(client: Client, user_auth_token: str):
+    """
+    arrange: log in a user.
```

Comment:
nitpicky suggestion, just to be more explicit :D
```suggestion
    arrange: log in as a non-admin user.
```

Reply:
Updated all these occurrences. Thx